arm/Microsoft.Network/azureFirewalls/deploy.bicep

```
@@ -215,6 +218,9 @@ resource azureFirewall 'Microsoft.Network/azureFirewalls@2021-02-01' = {
     additionalProperties: {
       'Network.DNS.EnableProxy': string(enableDnsProxy)
     }
+    firewallPolicy: {
+      id: empty(firewallPolicyIResourceId) ? null : firewallPolicyIResourceId
```

```suggestion
      id: !empty(firewallPolicyIResourceId) ? firewallPolicyIResourceId : null
```

Just for readibility please flip

---

If we do it like that, would the template remove an existing firewall policy if we don't provide one with the deployment? I'm thinking that maybe it's better to have a condition for the entire object. For example:

```bicep
firewallPolicy: !empty(firewallPolicyIResourceId) ? {
      id:  firewallPolicyIResourceId
} : null
``` 

However, I guess it is up to testing whether this is actually the case

---

changed

---

Did you by any chance test it?

---

test failed. Removed the property (reverted back to old one)

---

Alright, thanks